content/en/index.md

```diff
@@ -26,7 +26,7 @@ A Sigstore client, such as Cosign, requests a certificate from Fulcio, a code-si
 
 You don’t have to manage signing keys, and Sigstore services never obtain your private key. The public key that a Sigstore client creates gets bound to the issued certificate, and the private key is discarded after a single signing.
 
-After the client signs the artifact, the artifact's digest, signature and certificate are persisted in Rekor, an immutable, append-only transparency ledger, so that signing events can be publicly audited. Identity owners can monitor the log to verify that their identity is being properly used. This also timestamps the signing event, so that the short-lived certificate can be later verified.
+After the client signs the artifact, the artifact's digest, signature and certificate are persisted in Rekor, an immutable, append-only transparency ledger, so that signing events can be publicly audited. Identity owners can monitor the log to verify that their identity is being properly used. This also timestamps the signing event, so that the short-lived certificate can be later verified. Cosign has the ability to verify.
```

Comment:
I don't think this sentence is needed, the next paragraph discusses verification.

Reply:
OK.   It looks clumsy to me anyway.  Removed.